metrics/getters.py

```diff
@@ -124,8 +124,24 @@ def log_path_to_sms(log_fp: str, log_parser) -> (List, bool):
     return [sm_before, sm_after], True
 
 
-test_passed = lambda case, sm: case in sm and sm[case] == TestStatus.PASSED.value
-
-test_failed = lambda case, sm: case not in sm or any(
-    [sm[case] == status for status in [TestStatus.FAILED.value, TestStatus.ERROR.value]]
-)
+# test_passed = lambda case, sm: case in sm and sm[case] == TestStatus.PASSED.value
+# overrode with correct function
+
+def test_passed(case, sm):
```

Comment:
there's a difference between our pytest output and the datasets output

# test_repr (view_tests.tests.test_debug.CallableSettingWrapperTests.test_repr) (our runner's output)
# test_repr(view_tests.tests.test_debug.CallableSettingWrapperTests) (the dataset's output)

Reply:
Thanks for pointing this out, it does look like there may be some incorrectly identified test names caused by bad parsing.

I will continue to resolve this in #43, thanks so much for the suggestion on parsing these tests - I will validate and incorporate them.